src/Microsoft.AspNet.Razor/RazorEditorParser.cs

```diff
@@ -131,8 +131,8 @@ public virtual PartialParseResult CheckForStructureChanges(TextChange change)
             if (change.NewBuffer == null)
             {
                 throw new ArgumentException(RazorResources.FormatStructure_Member_CannotBeNull(
-                                                          "Buffer",
-                                                          "TextChange"), "change");
+                                                          nameof(change.NewBuffer),
```

Comment:
Was `Buffer` actually meant here or just an example of why the `nameof` operator is awesome?